src/alfasim_sdk/_internal/alfacase/generate_schema.py

```diff
@@ -158,6 +183,14 @@ def scalar_to_alfacase_schema(type_: type, indent: int) -> str:
     return 'Map({"value": Float(), "unit": Str()})'
 
 
+def is_scalar_expression(type_: type) -> bool:
+    return inspect.isclass(type_) and issubclass(type_, ScalarExpression)
```

Comment:
Can't this simply be `return type_ is ScalarExpression`? we don't have subclasses of `ScalarExpression`.